nx/lib/nx/tensor.ex

```diff
@@ -210,48 +210,68 @@ defmodule Nx.Tensor do
     raise "Access.pop/2 is not yet supported by Nx.Tensor"
   end
 
-  defimpl Inspect do
+  defimpl Inspect, for: Nx.Tensor do
     import Inspect.Algebra
 
-    def inspect(
-          %{shape: shape, names: names, type: type, vectorized_axes: vectorized_axes} = tensor,
-          opts
-        ) do
+    defp format_stats(tensor) do
+      n = Nx.size(tensor)
+      min = tensor |> Nx.reduce_min() |> Nx.to_number() |> :erlang.float()
```

Comment:
There's another conceptual problem here, im which Nx.Defn.Expr tensors will just break completely if this option is set.

If we decide to move forward with this, this mode should be implemented in the Nx.Backend inspect callback instead of if being placed here.

Reply:
This implementation is also not good if the tensor type is not float.